packages/ui/src/styles.ts

```diff
@@ -17,6 +17,14 @@ export const globalStyles = css`
     display: flow-root;
   }
 
+  .dark {
+    .iden-icon {
```

Comment:
I believe the `Identicon` component has a default class `ui--IdentityIcon` so we could use it for styling without having to add an extra class here. Moreover, this would force us to add the same class when ever we use this component in other place.